packages/client/hmi-client/src/workflow/tera-simulation-workflow.vue

```diff
@@ -82,38 +82,40 @@
 						:node="node"
 						@append-input-port="(event) => appendInputPort(node, event)"
 					/>
-					<tera-simulate-julia-node
+					<tera-simulate-node-julia
```

Comment:
Why the `node` is not at the end of the name? This is really weird

Reply:
@Tom-Szendrey 's convention, just following it.